### DISCUSSION TITLE
It won't let me submit my ID to apply for Teacher Benefits

### DISCUSSION THREAD

**Discussion by rafa8l** [community/community#32227] [GitHub Education]
I'm a multimedia teacher in a school in Brazil. Working mostly with interactive systems and animation.
I submitted my teacher's ID and the Github system keeps telling me that "to apply as a Student" the link is another, and/or to complete my profile (full name and bio), but I've done so.
What should I do?

**Comment by 3kh0** [comment]
You can contact [GitHub support here](https://support.github.com/) and open a support ticket where a support agent will help you!

*Mark this as the answer if I helped your problem*
